data/map/stairs.yml

```
@@ -351,21 +479,233 @@
     plane: 2
   delta:
     plane: -1
-# Lumbridge general store
-- id: 45481
+# Lumbridge north
+- id: 52833
   option: Climb-up
   tile:
-    x: 3215
-    y: 3239
+    x: 3140
+    y: 3305
   delta:
-    x: -3
+    plane: 1 
+- id: 52834
+  option: Climb-down
+  tile:
+    x: 3140
+    y: 3305
     plane: 1
-- id: 45482
+  delta:
+    plane: -1
+- id: 36795
+  option: Climb-up
+  tile:
+    x: 3164
+    y: 3307
+  delta:
+    plane: 1 
+- id: 36796
+  option: Climb-up
+  tile:
+    x: 3164
+    y: 3307
+    plane: 1
+  delta:
+    plane: 1     
+- id: 36796
   option: Climb-down
   tile:
-    x: 3215
-    y: 3239
+    x: 3164
+    y: 3307
     plane: 1
   delta:
-    x: 3
-    plane: -1
\ No newline at end of file
+    plane: -1
+- id: 36797
+  option: Climb-down
+  tile:
+    x: 3164
+    y: 3307
+    plane: 2
+  delta:
+    plane: -1    
+#wizard tower
+- id: 2147
+  option: Climb-down
+  tile:
+    x: 3104
+    y: 3162
+  delta:
```

```suggestion
  to:
```
Using the ladder from 3104, 3163 puts you into a wall

---

doesn't it need  x: 3104 y: 9576 after the to:

---

Yep it would